### DISCUSSION TITLE
Open Source Mirrors

### DISCUSSION THREAD

**Discussion by everything411** [LingmoOS/Community#23] [Ideas]
Does LingmoOS have mirrors for downloading? Does LingmoOS provide `rsync` service for mirroring? What is the current size of the repository?

**Comment by elysia-best** [comment]
@lingmo-dream is responsible for maintaining the archives. @lingmo-dream have a look at this?

I only maintain a nightly repo for testing, see https://github.com/LingmoOS/lingmo-nightly-build
